src/test/java/com/google/cloud/tools/managedcloudsdk/components/WindowsBundledPythonCopierTestHelper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Google Inc.
+ * Copyright 2018 Google Inc.
```

Comment:
Ditto year

Reply:
I think this one is fine, It's a new file that for some reason git thinks is a version of that other file that was removed.